packages/executor/src/services/UserOpValidation/validators/safe.ts

```diff
@@ -255,15 +256,18 @@ export class SafeValidationService {
         );
       }
 
-      const whitelist: IWhitelistedEntity | undefined =
-        WhitelistedEntities[entityTitle as keyof IWhitelistedEntities];
+      const whitelist =
+        this.networkConfig.whitelistedEntities[
+          entityTitle as keyof IWhitelistedEntities
+        ];
       if (
         entityAddr &&
         whitelist != null &&
         // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
-        whitelist[this.chainId] &&
-        whitelist[this.chainId]!.some(
-          (addr) => addr === ethers.utils.getAddress(entityAddr)
+        whitelist.some(
```

Comment:
Should we also check if the whitelisted address is a smart contract?

Reply:
no, the account might not be created yet during validation